stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/channels/ChannelOptions.kt

```diff
@@ -98,7 +98,7 @@ public fun buildDefaultChannelOptionsState(
 ): List<ChannelOptionState> {
     val canLeaveChannel = !selectedChannel.isDistinct()
     val canDeleteChannel = channelMembers.firstOrNull { it.user.id == currentUser?.id }
-        ?.role
+        ?.channelRole
         ?.let { it == "admin" || it == "owner" }
```

Comment:
I think `channelRole` has different values comparing to `role`. You can check details [here](https://getstream.io/chat/docs/react/legacy_permissions/#channel-user-roles)

Reply:
I think `admin` still works, but `owner` doesn't exist. So just the admin check should be fine I think